Formula/t/task@2.rb

```diff
@@ -0,0 +1,54 @@
+class TaskAT2 < Formula
+  desc "Feature-rich console based todo list manager"
+  homepage "https://taskwarrior.org/"
+  url "https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v2.6.2/task-2.6.2.tar.gz"
+  sha256 "b1d3a7f000cd0fd60640670064e0e001613c9e1cb2242b9b3a9066c78862cfec"
+  license "MIT"
+  revision 1
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  bottle do
+    rebuild 1
+    sha256                               arm64_sonoma:   "50c3572e274b41419876401235d67710754821708fe334d871d2b9cd116a8b35"
+    sha256                               arm64_ventura:  "11ca4562ebc4ea8273f0cab7ef18ec41980c83a373f4b6369b898ee19bea15eb"
+    sha256                               arm64_monterey: "2c1d0eb0f0522bbf4cdd0dea43c394afaceec873d58c609b211f6450cd09421d"
+    sha256                               arm64_big_sur:  "c2342e1543ea07eba65ab7bef36e25a0e463e6baeab6e562bbcb4a572faa89e7"
+    sha256                               sonoma:         "317a2c205a7d616e4773cc130bb3ecdf1b851f3b6e4311627cd558ae41a7517d"
+    sha256                               ventura:        "ffde36684ca7b6ed1e01327fd2aeaadb588aeb06125065cd8781d4b5ea630374"
+    sha256                               monterey:       "72cc6c37f541104645ff0d7f952abc7e3acf80cf2e91dd32eceecb80502baf53"
+    sha256                               big_sur:        "7671bf2e4b715cffd7895b97ff049295bea6faee8f4028c7c8c78be2f4ab8c66"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "53cb397182b40cbad438aefbe4fb38e505f0eac4b2999dd38ffbb8aad3d12f86"
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "cmake" => :build
```

Comment:
```suggestion
  keg_only :versioned_formula

  disable! date: "2025-04-30", because: :unmaintained

  depends_on "cmake" => :build
```

Reply:
Ah I did not know about `disable!`, thank you!